packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/StoreKit2/StoreKit2Translators.swift

```diff
@@ -188,14 +188,15 @@ extension Product.PurchaseResult {
 extension Transaction {
   func convertToPigeon(restoring: Bool = false) -> SK2TransactionMessage {
 
-    let dateFromatter: DateFormatter = DateFormatter()
-    dateFromatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
+    let dateFormatter: DateFormatter = DateFormatter()
```

Comment:
nit: `let dateFormatter = DateFormatter()`

Reply:
Sure, will update shortly. Just keep in mind I am not a swift dev, so snippets like this one are really helpful!

Reply:
Done